### ISSUE TITLE
Sync this Repo with the Launchpad Repository

### ISSUE THREAD

**Issue by dviererbe** [canonical/ubuntu-packaging-guide#1]
## Description
The `main` branch is a clone of [the Repository used to create the current packaging Guide](https://code.launchpad.net/~ubuntu-packaging-guide-team/+junk/update-packaging-guide).
*Note: it has already been converted to [git](https://code.launchpad.net/~ubuntu-packaging-guide-team/ubuntu-packaging-guide/+git/main).*

The plan is to automatically sync the main branch with the git repository on launchpad. 

One unanswered question is, if this GitHub Repository is considered up- or downstream in relation to the Launchpad git Repository.
(*My personal opinion is that this should be considered upstream as this will be the deployed version to Read the Docs*)

This issue should track that effort.

## TODO
- [ ] Get write access to the Launchpad git Repository for a bot
- [ ] Setup GitHub Actions Workflow to sync the Repositories

**Comment by lgp171188** [comment]
@dviererbe, Launchpad has a feature to automatically mirror Git repositories from other Git hosting services like GitHub. The instructions to do this are documented in https://help.launchpad.net/Code/Git#Mirroring_repositories_from_other_sites. You do not have to set up GitHub Actions to do this sync. I am happy to answer if you have any questions regarding this.

**Comment by dviererbe** [comment]
That is nice to know, thanks!

> In either case, Launchpad will mirror the whole repository from the remote site, and will keep its copy up to date regularly. **You won't be able to push directly to the imported repository on Launchpad**, but you can create another repository in the same project and push branches to that, and even create merge proposals if you want (though you may have to tell the upstream maintainer about them separately!). You can create source package recipes or Snap packages based on branches in the imported repository.
> 
> Please note that Launchpad can only mirror public repositories.

I had hoped that we can also mirror Merge Proposals/Pull Requests and Bugs/Issues, but this has a much more reasonable effort/benefit ratio.

TL;DR: I will go forward with this.

**Comment by dviererbe** [comment]
Closing this as this repository is slowly replaced by the [Ubuntu project docs](https://github.com/ubuntu/ubuntu-project-docs).
